LICENSE.md

```diff
@@ -1,3 +1,5 @@
+# part 1: Sam is the best 
```

Comment:
```suggestion
# part 1: Sam is the best 
#what about Sam I am
```

Reply:
no reaction

Reply:
good

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](http://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
fix

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient, Inelegant, Not reusable